ons_alpha/static_src/sass/components/design_system_overrides/_container.scss

```diff
@@ -0,0 +1,4 @@
+// overrides to ons container styles
+.ons-container {
+    max-width: 1024px;
```

Comment:
I realised the new design had a slightly different maximum width on all the container classes

Reply:
Now tweaked to account for the padding